lib/internal/Magento/Framework/App/FeedImporterInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Framework\App;
+
+interface FeedImporterInterface
+{
+
+    /**
+     * @throws \Magento\Framework\Exception\FeedImporterException
+     * @param  array  $data
+     * @param  string $format
+     * @return FeedInterface
+     */
+    public function importArray(array $data, $format = 'atom');
```

Comment:
It's not always a good Idea to replicate contracts from Zend. Because sometimes we would provide a wider contract than Zend does, and sometimes their contracts are not so ideal.
Above I described my idea what we can do with this